src/sentry/data_export/processors/issues_by_tag.py

```diff
@@ -97,7 +97,7 @@ def serialize_row(item, key):
         }
         if key == "user":
             euser = item._eventuser
-            result["id"] = euser.ident if euser else ""
+            result["id"] = euser.user_id if euser else ""
```

Comment:
Why do we stop using ident here?

Reply:
Hmm nm, I remember your comment here:
> @wedamija the dataclass's id is to mirror the EventUser model id column. user_id in the dataclass will replace the ident column in EventUser

Although now that I'm seeing it it is a little confusing to call it `user_id` since it implies an id from the `User` model. Should this at least be `user_ident` or something?